### ISSUE TITLE
Deployment/Removal error: that.sdk[service] is not a constructor

### ISSUE THREAD

**Issue by sime** [serverless/serverless-websockets-plugin#10]
I get the following error when trying to deploy/remove a stack with the websockets plugin. I haven't tried to do any functionality yet, I just want to see it deployed.

```
Serverless: Stack removal finished...
  Type Error ---------------------------------------------
  that.sdk[service] is not a constructor
     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.
```

```yaml
service: skippy-websockets

plugins:
  - serverless-websockets-plugin

provider:
  name: aws
  runtime: nodejs8.10
  region: eu-west-1

functions:
  connectionHandler:
    handler: handler.hello
    events:
      - websocket:
          routeKey: $connect
      - websocket:
          routeKey: $disconnect

  defaultHandler:
    handler: handler.hello
    events:
      - websocket:
          routeKey: $default

  actionHandler:
    handler: handler.hello
    events:
      - websocket:
          routeKey: myAction
```

```javascript
'use strict';

module.exports.hello = async (event, context) => {
  return {
    statusCode: 200,
    body: JSON.stringify({
      message: 'Go Serverless v1.0! Your function executed successfully!',
      input: event,
    }),
  };

  // Use this code if you don't use the http event with the LAMBDA-PROXY integration
  // return { message: 'Go Serverless v1.0! Your function executed successfully!', event };
};
```

**Comment by sime** [comment]
Got the same error whilst trying to run example application in `example/`

```
     Node Version:           8.12.0
     Serverless Version:     1.34.1
```

**Comment by sime** [comment]
Seems to be resolved by installing 1.35.1

**Comment by eahefnawy** [comment]
Thanks for reporting @sime ... because AWS recently introduced this feature, it has also introduced a completely new APIG Service in the SDK, which requires an update to the aws-sdk, and of course a framework update. That should explain the `that.sdk[service] is not a constructor` error.
